src/r_data.c

```diff
@@ -852,7 +852,7 @@ static byte invul_orig[256];
 
 void R_InvulMode(void)
 {
-  if (colormaps == NULL)
+  if (colormaps == NULL || beta_emulation)
```

Comment:
Should the menu switch get disabled as well?

Reply:
We could introduce the `S_BETA` flag and disable a lot of things that don't work in beta mode, but I don't think it's worth it. What if we remove beta emulation altogether?

Reply:
> a lot of things that don't work in beta mode

What would that be?

Reply:
> > a lot of things that don't work in beta mode
> 
> What would that be?

Complevels, "Preferences" tab in the "Weapons" menu, screen flashes. Probably more, I can't remember.

Reply:
Direct aiming for missiles.

Reply:
So should we disable the menu items for beta mode? I don't think it helps much. Currently, if users want to experience beta mode, they have to do the following:
1. Download mbf.zip for BETAGRPH.WAD and BETALEVL.WAD
2. Run the game with this command line: `woof -beta -complevel mbf -iwad doom.wad -file BETALEVL.WAD`

At this point it is easier (and more fun) to run press beta in DosBox.